python/CMakeLists.txt

```diff
@@ -38,6 +38,13 @@ import sysconfig;
 print(sysconfig.get_config_var('LINKFORSHARED'), end = '')"
     OUTPUT_VARIABLE PY_LINKFORSHARED_CONFIG)
 
+if(DEFINED ENV{PYCRYPTOSAT_INSTALL_PATH})
+    set(INSTALL_PREFIX "--prefix=$ENV{PYCRYPTOSAT_INSTALL_PATH}")
+elseif(DEFINED ENV{VIRTUAL_ENV})
+    set(INSTALL_PREFIX "")
```

Comment:
Virtual environments define the `VIRTUAL_ENV` variable which define their root.

We might be able to pass `--prefix=${VIRTUAL_ENV}`, but it makes sense to let `setup.py` do the right thing for us; this will go into the python virtual environment:

```
VERSION
    5.6.8

FILE
    /home/user/venv3/lib/python3.6/site-packages/pycryptosat.cpython-36m-x86_64-linux-gnu.so
```